pkg/arvo/app/btc-node-hook.hoon

```diff
@@ -0,0 +1,373 @@
+          ?=([%abort-rescan *] act)
+          ?=([%add-multisig-address *] act)
+          ?=([%backup-wallet *] act)
+          ?=([%bump-fee *] act)
+          ?=([%dump-privkey *] act)
+          ?=([%dump-wallet *] act)
+          ?=([%encrypt-wallet *] act)
+          ?=([%fund-raw-transaction *] act)
+          ?=([%get-balance *] act)
+          ?=([%get-addresses-by-label *] act)
+          ?=([%get-address-info *] act)
+          ?=([%get-new-address *] act)
+          ?=([%get-raw-change-address *] act)
+          ?=([%get-received-by-address *] act)
+          ?=([%get-received-by-label *] act)
+          ?=([%get-transaction *] act)
+          ?=([%get-unconfirmed-balance *] act)
+          ?=([%get-wallet-info *] act)
+          ?=([%import-address *] act)
+          ?=([%import-multi *] act)
+          ?=([%import-privkey *] act)
+          ?=([%import-pruned-funds *] act)
+          ?=([%import-pubkey *] act)
+          ?=([%import-wallet *] act)
+          ?=([%key-pool-refill *] act)
+          ?=([%list-address-groupings *] act)
+          ?=([%list-labels *] act)
+          ?=([%list-lock-unspent *] act)
+          ?=([%list-received-by-address *] act)
+          ?=([%list-received-by-label *] act)
+          ?=([%lists-in-ceblock *] act)
+          ?=([%list-transactions *] act)
+          ?=([%list-unspent *] act)
+          ?=([%lock-unspent *] act)
+          ?=([%remove-pruned-funds *] act)
+          ?=([%rescan-blockchain *] act)
+          ?=([%send-many *] act)
+          ?=([%send-to-address *] act)
+          ?=([%set-hd-seed *] act)
+          ?=([%set-label *] act)
+          ?=([%set-tx-fee *] act)
+          ?=([%sign-message *] act)
+          ?=([%sign-raw-transaction-with-wallet *] act)
+          ?=([%wallet-create-fundedpsbt *] act)
+          ?=([%wallet-lock *] act)
+          ?=([%wallet-passphrase *] act)
+          ?=([%wallet-passphrase-change *] act)
+          ?=([%wallet-process-psbt *] act)
+      ==
+    endpoint
+  ?:  ?=([?(%dump-wallet %import-wallet) filename=@t] act)
+    (crip "{(trip endpoint)}wallet/{(trip filename.act)}")
+  ::  FIXME: fails when default-wallet is '' and more than 1 wallet
+  ::  has been created
+  ::
+  ::    url='http://127.0.0.1:18443/wallet/'
+  ::
+  ::  although this example works with curl:
+  ::
+  :: curl --user XXX:YYY
+  ::      --data-binary '{
+  ::        "jsonrpc": "1.0",
+  ::        "id":"curltest",
+  ::        "method": "getwalletinfo",
+  ::        "params": []
+  ::      }'
+  ::      -H 'content-type: text/plain;'
+  ::      http://127.0.0.1:18443/wallet/
+  ::
+  ::  A %switch-wallet command needs to be issued against the store app
+  ::  after a wallet is created, in order to use it.
+  ::
+  ::  e.g.  > :btc-node-store|command [%switch-wallet 'local']
+  ::
+  ?:  (lte n-wallets 1)
+    endpoint
+  %-  crip  %+  weld
+    "{(trip endpoint)}wallet/"
+  ?:  =('' default-wallet)  ~
+  "{(trip default-wallet)}"
```

Comment:
Removing the `?:` here is fine, since the end result is the same. (And then you get to remove the weld too. You could `:((cury cat 3) endpoint 'wallet/' default-wallet)` if you don't want to go back and forth between cord and tape, I guess.)